nixos/modules/programs/wayland/wayland-session.nix

```diff
@@ -1,8 +1,8 @@
 {
   lib,
   pkgs,
-  xwayland ? true,
-  wlr-portal ? true,
+  enable-xwayland ? true,
+  enable-wlr-portal ? true,
```

Comment:
I would expect them to be named like this based on most other options we have

```
  enableXwayland ? true,
  enableWlrPortal ? true,
```

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/315827/commits/de777188870259d3a1633a41dad963cc19bee6a1